system/Validation/Rules.php

```diff
@@ -86,7 +86,11 @@ public function greater_than_equal_to(?string $str, string $min): bool
     public function is_not_unique(?string $str, string $field, array $data): bool
```

Comment:
I was trying something else, I removed "?string" and it worked.

`public function is_not_unique($str, string $field, array $data): bool`